### ISSUE TITLE
Windows Defender detected Win32/Uwamson.A!ml malware in OldUnreal-UTPatch469b-Windows.zip

### ISSUE THREAD

**Issue by craiglandis** [OldUnreal/UnrealTournamentPatches#536]
Windows Defender detects Win32/Uwamson.A!ml in OldUnreal-UTPatch469b-Windows.zip - 

![Windows_Defender_potentially_unwanted_app_found_Win32_UwamsonAml_OldUnreal-UTPatch469b-Windows zip](https://user-images.githubusercontent.com/2888449/132112567-da84b1c9-1197-4e67-babe-618d9cc86b1e.png)

https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Program%3aWin32%2fUwamson.A!ml&threatid=250070

**Comment by SeriousBuggie** [comment]
Explanation: #15 and #20. Also this is duplicate.

Please read [Readme.md](https://github.com/OldUnreal/UnrealTournamentPatches#readme):

> Malware Warnings
> 
> Our Windows and macOS binaries are digitally signed, but malware/virus scanners may occasionally still flag them as potential malware because our signing certificate is still building up reputation. Additional information can be found here.

**Comment by stijn-volckaert** [comment]
The Windows Defender detection has been removed. The patch currently gets flagged as potentially unwanted software by 15/60 scanners on virustotal. The main issue is of course that our patch, as well as the digital certificate I used to sign all exe and dll files, is relatively unknown to these scanners. Unfortunately, we simply do not have the resources to remove these false positive detections whenever they pop up.
